Gems/PhysX/Code/Include/PhysX/Joint/Configuration/PhysXJointConfiguration.h

```diff
@@ -84,6 +84,22 @@ namespace PhysX
         float m_tolerance = 0.1f; ///< Distance from the joint at which limits becomes enforced. Used if limit is hard.
     };
 
+    struct JointMotorProperties
```

Comment:
question: This looks very similar to `EditorJointMotorConfig`, could they share the same internal struct or could we just pick one? Just curious why we need both and there's a chance they could get out of sync (the naming variable names are also super similar but not quite the same which could cause confusion).

Reply:
Removed EditorJointMotorConfig class.